demo/prepare-devnet.sh

```diff
@@ -8,6 +8,8 @@ BASEDIR=$(realpath $(dirname $(realpath $0))/..)
 TARGETDIR="devnet"
 
 cp -afT "$BASEDIR/local-cluster/config" "$TARGETDIR"
+find $TARGETDIR -type f -exec chmod 0400 {} \;
```

Comment:
Don't we need only the signing keys to be `400` :thinking: ? I guess it doesn't hurt to also chmod the others..